python/py-virtualenv/Portfile

```diff
@@ -19,24 +19,44 @@ long_description    virtualenv is a tool to create isolated Python \
                     environments.
 
 homepage            https://virtualenv.pypa.io
-master_sites        pypi:v/${python.rootname}
-distname            ${python.rootname}-${version}
 
```

Comment:
where do you specify the build dependencies from you're purposefully ignoring the other build dependencies? https://github.com/pypa/virtualenv/blob/master/pyproject.toml#L3-L5?

Reply:
Ehi thanks to take time to review this.

I'm working on this... But it looks like `virtualenv` needs `py-setuptools_scm[toml]`. How do I specify in a Portfile the `toml` extra?

Reply:
I guess I just provide the dependency... I'm dumb! Sorry for asking.